test/functions/iterators/elliot/dynamicIters/numTasksDynamicIters.neg.good

```diff
@@ -0,0 +1,13 @@
+numTasksDynamicIters.chpl:11: warning: 'numTasks' < 0, defaulting to '1' tasks
```

Comment:
"defaulting to 1 task" (no single-quotes and "task" instead of "tasks") would be much prettier.

Reply:
That's fine.  The quotes seem odd and I think are worth removing, but based on experience I know messages of the form "... 1 \<thing\>s ..." bother me more than anyone else.